.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: go
+
+go:
+  - 1.8.x
```

Comment:
If I merge this to master I can just switch a build on in DockerHub initially; is it working for you otherwise?